agent/src/main.cpp

```diff
@@ -250,8 +250,10 @@ int main(int argc, char **argv)
     } catch (ConfigError &error) {
         log_error() << "Could not load configuration";
         return 1;
-    } catch (ReturnCode failure) {
-        log_error() << "Agent initialization failed";
+    } catch (SoftwareContainerError &error) {
+        // Currently only case where any other exception can occur is during Agent init
```

Comment:
Maybe it is an unnecessary comment, but since I catch on the top level on a very large block of code (which does more that just initialize the Agent) it felt strange to me add the text "agent init failed". But maybe I don't even need that text? Not sure.